src/app/papers.ts

```diff
@@ -1,6 +1,48 @@
+  citation: '@inproceedings{10.1145/3652588.3663321,\n\
+    author = {Erhard, Julian and Schinabeck, Johanna Franziska and Schwarz, Michael and Seidl, Helmut},\n\
+    title = {When to Stop Going Down the Rabbit Hole: Taming Context-Sensitivity on the Fly},\n\
+    year = {2024},\n\
+    isbn = {9798400706219},\n\
+    publisher = {Association for Computing Machinery},\n\
+    address = {New York, NY, USA},\n\
+    url = {https://doi.org/10.1145/3652588.3663321},\n\
+    doi = {10.1145/3652588.3663321},\n\
+    abstract = {Context-sensitive analysis of programs containing recursive procedures may be expensive, in particular, when using expressive domains, rendering the set of possible contexts large or even infinite. Here, we present a general framework for context-sensitivity that allows formalizing not only known approaches such as full context or call strings but also combinations of these. We propose three generic lifters in this framework to bound the number of encountered contexts on the fly. These lifters are implemented within the abstract interpreter Goblint and compared to existing approaches to context-sensitivity on the SV-COMP benchmark suite. On a subset of recursive benchmarks, all proposed lifters manage to reduce the number of stack overflows and timeouts compared to a full context approach, with one of them improving the number of correct verdicts by 31\% and showing promising results on the considered SV-COMP categories.},\n\
+    booktitle = {Proceedings of the 13th ACM SIGPLAN International Workshop on the State Of the Art in Program Analysis},\n\
+    pages = {35-44},\n\
+    numpages = {10},\n\
+    keywords = {abstract interpretation, context-sensitive analysis, software verification, static program analysis},\n\
+    location = {Copenhagen, Denmark},\n\
+    series = {SOAP 2024}\n\
+    }',
+  authors: ['J. Erhard', 'J. F. Schinabeck', 'M. Schwarz', 'H. Seidl']
+  },
+  {
+  link: 'https://link.springer.com/article/10.1007/s10009-024-00755-0',
+  title: 'Non-numerical weakly relational domains',
+  conference: 'STTT',
+  year: 2024,
+  description: 'The weakly relational domain of Octagons offers a decent compromise between precision and efficiency for numerical properties. Here, we are concerned with the construction of non-numerical relational domains. We provide a general construction of weakly relational domains, which we exemplify with an extension of constant propagation by disjunctions.',
+  citation: '@article{10.1007/s10009-024-00755-0,\n\
+    author = {Seidl, Helmut and Erhard, Julian and Tilscher, Sarah and Schwarz, Michael},\n\
+    year = {2024},\n\
+    month = {06},\n\
+    pages = {1-16},\n\
+    title = {Non-numerical weakly relational domains},\n\
+    journal = {International Journal on Software Tools for Technology Transfer},\n\
+    doi = {10.1007/s10009-024-00755-0}\n\
+    }',
+  authors: ['H. Seidl', 'J. Erhard', 'S. Tilscher', 'M. Schwarz']
```

Comment:
Do those not need to be escaped for the copy citation button to work?

Reply:
It seems not. At least when I serve it locally, the copying works as intended. 

For the 'Correctness Witness Validation by Abstract Interpretation', article citation, they are not escaped either.